src/main/asciidoc/batch/file-ingest-sftp/kubernetes.adoc

```diff
@@ -0,0 +1,278 @@
+Successfully registered application 'sink:task-launcher'
+----
++
+. Register and create the file ingest task:
+[source,console,options=nowrap]
+dataflow:>app register --name fileIngest --type task --uri docker:springcloud/ingest
+Successfully registered application 'task:fileIngest'
+dataflow:>task create fileIngestTask --definition fileIngest
+Created new task 'fileIngestTask'
++
+. Create and deploy the stream
++
+Now lets create and deploy the stream.
+Once deployed, the stream will start polling the SFTP server and, when new files arrive, launch the batch job.
++
+NOTE: Replace `<user>`, '<pass>`, and `<host>` below.
+The `<host>` is the SFTP server host, `<user>` and `<password>` values are the credentials for the remote user. 
+Additionally, replace `--spring.cloud.dataflow.client.server-uri=http://<dataflow-server-ip>` with the Cluster IP (External IP should work as well) of your dataflow server, as shown by `kubectl get svc/scdf-server`. 
+The default Data Flow server credentials are `user` and `password`.
++
+Notice how we use the Kubernetes Persistent Volume Claim(PVC) resource that we created earlier. 
+In the stream definition, the PVC and the associated Volume Mount are passed to the task via `--task.launch.request.deployment-properties`. 
+The `deployer.\*.kubernetes...` properties provide native Kubernetes specs as JSON to instruct the Data Flow server's deployer to add this configuration to the container configuration for the pod that will run the batch job. 
+We mount the NFS shared directory that we configured in the `nfs` Persistent Volume(PV) as `/staging` in the pod's local file system. The `nfs` PVC allows the pod to allocate space on the PV.
+The corresponding configuration, targeting the `sftp` source is used to deploy the stream.
+This enables the `sftp` source to share NFS mounted files with the launched task.
++
+NOTE: The deployment property `deployer.*.kubernetes.bootMajorVersion=2` tells the deployer to configure the liveness and readiness probes for Spring Boot 2.x applications, as Spring Boot actuator endpoints change in version 2. This will likely become the default in a future SCDF release.
++
+[source,console,options=nowrap]
+----
+dataflow:>stream create inboundSftp --definition "sftp --host=<host> --username=<user> --password=<password> --allow-unknown-keys=true --remote-dir=/remote-files --local-dir=/staging/shared-files --task.launch.request.taskName=fileIngestTask --task.launch.request.deployment-properties='deployer.*.kubernetes.volumes=[{\"name\":\"staging\",\"persistentVolumeClaim\":{\"claimName\":\"nfs\"}}],deployer.*.kubernetes.volumeMounts=[{\"mountPath\":\"/staging\",\"name\":\"staging\"}]'| task-launcher --spring.cloud.dataflow.client.server-uri=http://<dataflow-server-ip> --spring.cloud.dataflow.client.authentication.basic.username=user --spring.cloud.dataflow.client.authentication.basic.password=password"
```

Comment:
It'd be better to move the stream definition up, so it is easy to reason through the described steps afterward.